spec/import_result_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'spec_helper'
+
+describe ActiveAdminImport::ImportResult do
+  context "failed_message" do
+    let(:import_result) { ActiveAdminImport::ImportResult.new }
+
+    before do
+      require'pry';binding.pry
+      Author.create(name: 'John', last_name: 'Doe')
+      Author.create(name: 'Jane', last_name: 'Roe')
+
+      @result = double \
+        failed_instances: [
+          Author.create(name: 'Jim', last_name: "Doe"), # {:last_name=>["has already been taken"]}
+          Author.create(name: nil,   last_name: 'Doe')  # {:name=>["can't be blank"], :last_name=>["has already been taken"]}
```

Comment:
Line is too long. [125/80]<br>Unnecessary spacing detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.